galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/OpenstackHttpClient.java

```diff
@@ -552,6 +554,7 @@ protected String getImageId(@NotNull String image) throws OpenstackManagerExcept
                 }
 
                 Images images = gson.fromJson(entity, Images.class);
+                logger.info(images);
```

Comment:
Are you expecting Images.toString() to give use sensible output ? or are you just trying to see the reference isn't null ? This will throw NPE if it is null I suspect?